docs/quickstart/testnet_on_digitalocean.md

```diff
@@ -0,0 +1,68 @@
+<!--
+order: 8
+-->
+
+# Deploy testnet to Digital Ocean
+
+Learn how to deploy testnet to Digital Ocean
+
+## Digital Ocean 
+
+### Account Setup
+
+Head over to https://www.digitalocean.com/ and create an account.
+
+DigitalOcean will want a public key that it can place on any Droplets we start, so that we can access them with a key that we know only we have.
+
+Let's create an SSH keypair now using `ssh-keygen -t rsa -b 4096` 
+
+This will ask you for a file where you want to save the key which you can call something like - digital-ocean-key.
+
+It'll also ask for a passphrase - feel free to set one if you wish or you could leave it empty. If you created it in the same folder as we've been working out of, you'll see two files - one called digital-ocean-key and one called digital-ocean-key.pub - these are respectively your private and public keys.
+
+In your DigitalOcean account, on the bottom left hand side, there is a link for 'Security'. Follow this link, and the next page will have an option to add an SSH key
+
+Click 'Add an SSH key' and you'll be presented with a dialog to enter your key. Simply copy the contents of your digital-ocean-key.pub into the large text box (you can get the contents printed to the terminal with cat digital-ocean-key.pub).
+
+### Create Droplet
+
+Once you've added your SSH key. click on the 'Droplets' link on the left, and then on the next page click 'Create Droplet'.
+
+On this page, you'll be presented with a number of options for configuring your DigitalOcean Droplet, including the distribution, the plan, the size/cost per month, region, and authentication. Feel free to choose whichever settings work best for you.
+
+Under Authentication, select 'SSH Key', and select which keys you would like to use (like the one you created in the last step). You can also name your Droplet if you wish. When you're finished, click 'Create Droplet' at the bottom.
+
+Wait a minute for your Droplet to start up. It'll appear under the 'Droplets' panel with a green dot next to it when it is up and ready. At this point, we're ready to connect to it.
+
+## Deploy to Droplet
+
+### Connect to Droplet
+
+Click on the started Droplet, and you'll see details about it. At the moment, we're interested in the IP address - this is the address that the Droplet is at on the internet.
+
+To access it, we'll need to connect to it using our previously created private key. From the same folder as that private key, run:
+
+```bash
+$ ssh -i digital-ocean-key root@<IP_ADDRESS>
+```
+Now you are connected to the droplet. 
+
+### Install Ethermint
+
+Clone and build Ethermint in the droplet using `git`:
+
+```bash
+git clone https://github.com/ChainSafe/ethermint.git
+cd ethermint
+make install
+```
+
+Check that the binaries have been successfuly installed:
+
+```bash
+ethermintd -h
+ethermintcli -h
+```
+### Start testnet
+
+Follow [Start Testnet](./testnet.md) to setup the testnet 
```

Comment:
update to the doc above and add `{hide}` at the end. See other docs for reference